styles/globals.css

```diff
@@ -41,6 +41,12 @@ html {
   background: rgba(46, 64, 90, 0.75);
 }
 
+.blur-opaque {
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+  backdrop-filter: blur(75px);
+  background: rgba(46, 64, 90);
```

Comment:
The `chapter-1` has an overall blue theme and having the RGB(46,64,90) works for this challenge but what about the other challenges that will be added in future, for example - [figma link](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=65%3A1012&t=yi7Typ86UjVL20qs-0), it has a red theme.

Reply:
The `blur-opaque` class is the "opaque" version of the `blur` class which is used by **layout.tsx** to display its background.
And similar to the `blur-opaque` class, the `blur` class also uses static color.

Once new chapters get introduced in the codebase, we can make both these classes have dynamic color properties.

Reply:
I'm just wondering why we're doing this via our `globals.css` file. Why not define a new color in our `tailwind.config.js` file and use the resulting tailwind class instead? If we then need to support a new color we just add that too and use its resulting tailwind class later on when we implement new challenges that have a different design.

Reply:
Addressed in ea822132cc3d13887c40e033b9b5d1605f6ed579

Defined new custom theme elements for `opacity` `backdropBlur` and `dropShadow` in **tailwind.config.js** and used them in place of the `blur` and `blur-opaque` classes.